tests/components/group/test_config_flow.py

```diff
@@ -549,9 +550,32 @@ async def test_config_flow_preview(
         "state": "unavailable",
     }
 
+    await client.send_json_auto_id(
```

Comment:
The option flow subscribed after the states were set so I changed the config flow test to unsub, set the states, than resub so it doesn't see the in between states